### PR TITLE
Issue #2961 Removing OKD 3.9.0 references from documentation

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

Fixes <place holder for #issue number>

Steps to test the pull request

  1. 
  2. 
  3. 
  4.

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
retest this please

**Comment by LalatenduMohanty** [comment]
retest this please

File: docs/source/using/experimental-features.adoc [hunk 1]
```diff
@@ -33,18 +33,17 @@ You can set the `MINISHIFT_ENABLE_EXPERIMENTAL` environment variable to enable t
...
+Using the experimental flag, to enable the Service Catalog in OKD 3.10 or in later versions, the command is:
 
 ----
 $ MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --extra-clusterup-flags "--enable=*,service-catalog"
 ----
 
+[NOTE]
+====
+The recommended way to enable service catalog is xref:../command-ref/minishift_openshift_component_add.adoc#[`minishift openshift component add`] command.
+====
```
Comment (praveenkumar) [inline docs/source/using/experimental-features.adoc]: 👍

File: docs/source/using/experimental-features.adoc [hunk 2]
```diff
@@ -33,18 +33,17 @@ You can set the `MINISHIFT_ENABLE_EXPERIMENTAL` environment variable to enable t
...
 Enables passing flags directly to `oc cluster up` that are not directly exposed in the {project} CLI.
 
-Using the experimental flag, to enable the Service Catalog in OKD 3.9, the command is:
-
-----
-$ MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --extra-clusterup-flags "--service-catalog"
-----
-
-For OKD 3.10.0 or later the command is:
+Using the experimental flag, to enable the Service Catalog in OKD 3.10 or in later versions, the command is:
```
Comment (praveenkumar) [inline docs/source/using/experimental-features.adoc]: 👍

**Comment by LalatenduMohanty** [comment]
The CI failures https://ci.centos.org/job/minishift-pr/5417/console and not related to the PR as it only contains doc changes.

**Pull request merged**